### PR TITLE
Use rpath for pypi artifact

### PR DESCRIPTION
**Pull request by kirklandsign**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -693,6 +693,14 @@ if(EXECUTORCH_BUILD_PYBIND)
                  BUILD_RPATH "@loader_path/../../../torch/lib"
                  INSTALL_RPATH "@loader_path/../../../torch/lib"
     )
+  else()
+    set_target_properties(
+      portable_lib
+      PROPERTIES # Assume that this library will be installed in
+                 # `site-packages/executorch/extension/pybindings`, and that
+                 # the custom_ops_aot_lib should be found with relative path.
+                 BUILD_RPATH "$ORIGIN/../../examplesmodels/llama2/custom_ops"
```
Comment (larryliu0820) [inline CMakeLists.txt:702]: typo

Comment (kirklandsign) [reply CMakeLists.txt]: Thank you

I need to see how to download the artifacts

Comment (kirklandsign) [reply CMakeLists.txt]: the workflow is only a dry run

**Review by larryliu0820 — APPROVED** [review]

File: CMakeLists.txt [hunk 2]
```diff
@@ -693,6 +693,14 @@ if(EXECUTORCH_BUILD_PYBIND)
                  BUILD_RPATH "@loader_path/../../../torch/lib"
                  INSTALL_RPATH "@loader_path/../../../torch/lib"
     )
+  else()
+    set_target_properties(
+      portable_lib
+      PROPERTIES # Assume that this library will be installed in
+                 # `site-packages/executorch/extension/pybindings`, and that
+                 # the custom_ops_aot_lib should be found with relative path.
+                 BUILD_RPATH "$ORIGIN/../../examples/models/llama2/custom_ops"
```
Comment (kirklandsign) [inline CMakeLists.txt:702]: Should probably be BUILD_RPATH "$ORIGIN:$ORIGIN/../../examples/models/llama2/custom_ops"
Comment

**Pull request merged**